pkg/sms/model.go

```diff
@@ -80,6 +81,15 @@ type Language struct {
 	LanguageCode string `json:"languageCode,omitempty"`
 }
 
+type RegionalOptions struct {
+	IndiaDlt *IndiaDltOptions `json:"indiaDlt,omitempty"`
```

Comment:
Hm... this one is a bit nit-picky. Go prefers acronyms (DLT in this case) to be all in same case (be that lower or capital): [reference](https://github.com/golang/go/wiki/CodeReviewComments#initialisms).
For example, see how `ContentTemplateID` field below has the `ID` part in all capital letters.

For this auto-generated code you can check out the `COMMON_ACRONYMS` constant in the `org.infobip.api.libs.processors.go.language.Identifier` class of the CIGLA project. It might be enough to add `"dlt"` string to that list.

If that change *doesn't fix* all these identifiers (so both struct name and the field name) than we can just ship it like this. If it fixes it I think it's worth it to keep the generated code looking more go-like.

Reply:
Updated variable names